rust/src/bin/apigen.rs

```
@@ -162,7 +197,10 @@ fn main_imp() -> Result<()> {
     let mut args = args.into_iter();
     while let Some(arg) = args.next() {
         match arg.as_str() {
-            "-h" | "--help" => println!("usage: {USAGE}"),
+            "-h" | "--help" => {
+                println!("usage: {USAGE}");
+                return Ok(());
```

Why would you silently ignore remaining arguments?